src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -79,19 +86,14 @@ final class MockTestStore: Store, AppConfigCaching, PrivacyPreservingProviding {
 
 	// MARK: - PrivacyPreservingProviding
 
-	var isPrivacyPreservingAnalyticsConsentGiven: Bool = false {
-		didSet {
-			currentRiskExposureMetadata = nil
-			previousRiskExposureMetadata = nil
-			userMetadata = nil
-			lastSubmittedPPAData = nil
-			lastAppReset = nil
-			lastSubmissionAnalytics = nil
-		}
-	}
+	var isPrivacyPreservingAnalyticsConsentGiven: Bool = true
```

Comment:
Shouldn't we align with the real store and keep the tinfoil attached, i.e. have `false` as the initial value?

Reply:
See answer above: I changed so the defaults match together